cmd/plasmacli/subcmd/eth/status.go

```diff
@@ -0,0 +1,44 @@
+package eth
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/config"
+	"github.com/FourthState/plasma-mvp-sidechain/eth"
+	"github.com/spf13/cobra"
+)
+// StatusCmd returns the current state of the eth connection (syncing, crashed etc)
+func StatusCmd() *cobra.Command {
+	return statuscmd
+}
+
+var statuscmd = &cobra.Command {
+	Use:   "status",
+	Short: "check state of eth connection",
+	Long: `returns current state of eth connection (syncing, crashed, etc)`,
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		// Get Conf
+		conf, err := config.ParseConfigFromViper()
+		if err != nil {
+			return fmt.Errorf("error retrieving config: { %s }", err)
+		}
+
+		// Initialize Eth Connection
+		client, err := eth.InitEthConn(conf.EthNodeURL)
+		if err != nil {
+			return fmt.Errorf("error initializing connection with client: { %s }", err)
+		}
+
+		// Check whether client synced
+		synced, err := client.Synced()
+		if err != nil {
+			return fmt.Errorf("error checking synced status: { %s }", err)
+		}
```

Comment:
Past this, I would not consider the following statement errors. Should switch these to normal print statements. Also you don't need to camel case statements that are meant to be consumed by the reader, "ethNode".

i.e
if (synced) print("live full node and synced")
else print("live full node. However the full node is not fully synced")

Reply:
should eth block height be printed to?